guides/rest_doc.md

```diff
@@ -741,6 +745,150 @@ kintoneUtility.rest.getFormLayout({
 });
 ```
 
+## <a name="postDeploy"> kintoneUtility.rest.postDeploy(param)
+* Deploy the apps to the production environments.
+
+#### Parameter
+
+| Name | Data type | Required | Description
+|:-----------|:------------:|:------------:|:------------
+| param | Object | Yes |  |
+| param.apps | Object | Yes | The array of the app information.
+| param.apps[].app | Number | Yes | The app ID.
+| param.revision | Number | Optional | The expected revision number. If the value does not match, an error will occur and the record will not be updated. If the value is not specified or is -1, the revision number will not be checked.
+| param.revert | Boolean | Optional | **Default:** `false`. Set `true` if you want to revert the deploy when some error occur during it.
+| param.isPreview | Boolean | Optional | **Default:** `false`. Set `true` if you want to get pre-live setting.
+| param.isGuest | Boolean | Optional | **Default:** `false`. Set `true` if the app belongs to a guest space.
+
+#### Response
+```
+Promise Object
+```
+#### Sample
+```js
+kintoneUtility.rest.postDeploy({
+  apps: [
+    { app: 542 },
+  ],
+}).then(function(response) {
+  console.log(response);
+}).catch(function(error) {
+  console.log(error);
+});
+```
+
+## <a name="getDeploy"> kintoneUtility.rest.getDeploy(param)
+* Get the status of the deployments of the apps.
+
+#### Parameter
+
+| Name | Data type | Required | Description
+|:-----------|:------------:|:------------:|:------------
+| param | Object | Yes |  |
+| param.apps | Number[] | Yes | The array of the ids of the apps.
+| param.isGuest | Boolean | Optional | **Default:** `false`. Set `true` if the app belongs to a guest space.
+
+#### Response
+```
+Promise Object
+```
+#### Sample
+```js
+kintoneUtility.rest.getDeploy({
+  apps: [ 542 ],
+}).then(function(response) {
+  console.log(response);
+}).catch(function(error) {
+  console.log(error);
+});
+```
+
+## <a name="getCustomize"> kintoneUtility.rest.getCustomize(param)
+* Get the information of JavaScrit/CSS files used to customize an app.
+
+#### Parameter
+
+| Name | Data type | Required | Description
+|:-----------|:------------:|:------------:|:------------
+| param | Object | Yes |  |
+| param.app | Number | Yes | The app ID.
+| param.isPreview | Boolean | Optional |  **Default:** `false`. Set `true` if you want to get pre-live setting.
+| param.isGuest | Boolean | Optional | **Default:** `false`. Set `true` if the app belongs to a guest space.
+
+#### Response
+```
+Promise Object
+```
+#### Sample
+```js
+kintoneUtility.rest.getCustomize({
```

Comment:
Because customize is a verb, so the function name should be changed
getCustomize => getCustomization